actions/play_youtube_playlist_MOD.js

```diff
@@ -37,22 +39,19 @@ module.exports = {
     const { Audio } = this.getDBM()
     const Mods = this.getMods()
     const url = this.evalMessage(data.url, cache)
-    const ytpl = Mods.require('ytpl')
-    // const moment = Mods.require('moment')
+    const maxvideos = this.evalMessage(data.maxvid, cache) || 250
+    const ytpl = Mods.require('ytpl') // be sure you have the latest YTPL, this was modified with 2.0.3 in mind
     const { msg } = cache
     const options = {}
```

Comment:
```suggestion
    const options = {
      watermark: 'highWaterMark: 1' // idk what this does, but the queue data has it, so i might as well add it in case someone needs it
    }
```